apl-core/pom.xml

```diff
@@ -219,6 +219,11 @@
       <artifactId>junit-platform-engine</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.skyscreamer</groupId>
```

Comment:
If that library is highly needed it should be put into list of copy exclusions in:
**Apollo/apl-exec/pom.xml**

```
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-dependency-plugin</artifactId>
...
            <configuration>
              <excludeTypes>zip</excludeTypes>
              <excludeArtifactIds>weld-junit5,weld-junit-common</excludeArtifactIds>
              <excludeGroupIds>org.junit,org.mockito,org.opentest4j,org.hibernate.javax.persistence</excludeGroupIds>

```

Reply:
It does not affect resulting /lib content, since maven-dependency-plugin does not copy any of <scope>test</scope> dependencies. But i will add it to exclusion list for consistency